test/contracts/Positions/BinaryOption.js

```diff
@@ -23,14 +23,14 @@ const {
 	convertToDecimals,
```

Comment:
The test file name is still BinaryOption.js, probably this should be changed as well